src/cli-sdk/src/config/definition.ts

```diff
@@ -545,3 +545,19 @@ export const definition = j
       description: 'Print helpful information',
     },
   })
+
+  .opt({
+    'expect-results': {
+      hint: 'value',
+      description: `When running \`vlt query\`, this option allows you to
+                    set a expected number of resulting items.
+
+                    Accepted values are numbers, booleans: true (to expect at
+                    least one result), false (to expect no results) and
+                    strings.
+
+                    Strings starting with \`>\`, \`<\`, \`>=\` or \`<=\`
+                    followed by a number can be used to check if the result
+                    is greater than or less than a specific number.`,
+    },
```

Comment:
Mixing boolean and string types is an antipattern. It would be the only boolean that is not a flag, gets set to `'true'` in the environment instead of `1` for true and `0` for false, etc.

It would be better to use `--expect-results=">0"` instead of `"true"`, which means the same thing, or `--expect-results=0` instead of `false`.

This definition should have a validation function, like:

```ts
validate: (value: string) => /^([<>]=?)?[0-9]+$/.test(value)
```

Then we can be ensured that no matter where the value comes from, it'll match the expected value.